plugins/ipam/dhcp/daemon.go

```diff
@@ -50,6 +50,12 @@ func newDHCP() *DHCP {
 	}
 }
 
+func generateClientID(containerID string, netName string, ifName string) string {
+	clientID := containerID + "/" + netName + "/" + ifName
```

Comment:
suppose we just make it one line as 
`return containerID + "/" + netName + "/" + ifName`

Reply:
done